testdata/templates/ce.metrics.tpl

```diff
@@ -9,7 +9,7 @@ headers:
   # to get the GUID, run: ibmcloud resource service-instance <NAME> --output json | jq -r '.[].guid'
   # https://cloud.ibm.com/docs/monitoring?topic=monitoring-mon-curl
   IBMInstanceID: {{ .GUID }}
-provider: IBM Cloud Code Engine Sysdig
+provider: ce
```

Comment:
@sriumcp Can you clarify the usage of this field? It should be `ce` to match the `providers` used in for example initializing the experiment, right?

Reply:
Yes. This is a descriptive tag for end users. Of course, we want to match providers in the chart values with this provider name.